vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManagerTest.java

```diff
@@ -115,5 +115,29 @@ public void test_deleteImagetBlockById_whenIdIsNull() throws BlockDoesNotExistEx
         Mockito.verify(imageBlockRepo, Mockito.never()).deleteById(imageBlockId);
 
     }
+    
+    @Test
+    public void test_adjustContentOrder_success() throws BlockDoesNotExistException {
+        String contentBlockId = "contentBlockId_2";
+        Optional<ContentBlock> contentBlockOptional = Optional.of(contentBlock);
+        when(contentBlockRepository.findById("contentBlockId_2")).thenReturn(contentBlockOptional);
+        managerToTest.adjustContentOrder(contentBlockId,Integer.valueOf(2));
+        assertEquals(Integer.valueOf(2), contentBlockOptional.get().getContentOrder());
+        Mockito.verify(contentBlockRepository).save(contentBlockOptional.get());
+    }
+
+    @Test(expected = BlockDoesNotExistException.class)
+    public void test_adjustContentOrder_forNonExistentId() throws BlockDoesNotExistException {
+        String contentBlockId = "notARealId";
+        Mockito.doThrow(BlockDoesNotExistException.class).when(contentBlockRepository).findById(contentBlockId);
+        managerToTest.adjustContentOrder(contentBlockId,Integer.valueOf(2));
+    }
+
+    @Test
+    public void test_adjustContentOrder_whenIdIsNull() throws BlockDoesNotExistException {
+        String contentBlockId = null;
+        managerToTest.adjustContentOrder(contentBlockId,Integer.valueOf(2));
+        Mockito.verify(contentBlockRepository, Mockito.never()).findById(contentBlockId);
```

Comment:
this isn't really worth testing.  even if the method would call `findById`, as long as no exception is thrown the test would pass. If anything, I would make sure save is never called as this would indicate data is being changed that shouldn't be changed.